gstcefaudiometa.cc

```diff
@@ -37,7 +37,7 @@ gst_buffer_add_cef_audio_meta (GstBuffer * buffer, GstBufferList *buffers)
 GType
 gst_cef_audio_meta_api_get_type (void)
 {
-  static volatile GType type;
+  static GType type;
```

Comment:
```
In file included from /usr/include/glib-2.0/glib/gthread.h:32,
                 from /usr/include/glib-2.0/glib/gasyncqueue.h:32,
                 from /usr/include/glib-2.0/glib.h:32,
                 from /usr/include/gstreamer-1.0/gst/gst.h:27,
                 from /home/reinismu/projects/gstcefsrc/gstcefaudiometa.h:4,
                 from /home/reinismu/projects/gstcefsrc/gstcefaudiometa.cc:1:
/home/reinismu/projects/gstcefsrc/gstcefaudiometa.cc: In function ‘GType gst_cef_audio_meta_api_get_type()’:
/usr/include/glib-2.0/glib/gatomic.h:117:19: error: argument 2 of ‘__atomic_load’ must not be a pointer to a ‘volatile’ type
  117 |     __atomic_load (gapg_temp_atomic, &gapg_temp_newval, __ATOMIC_SEQ_CST); \
      |     ~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
I got this error when compiling

Did a bit of googling and found https://github.com/vmware/open-vm-tools/commit/b2c8baeaa8ac365e1445f941cf1b80999ed89a9d

Reply:
https://gitlab.freedesktop.org/gstreamer/gst-plugins-base/-/commit/98249a57dbd57dffbda12bfa9cb841fa41b28bbd

Reply:
damn it, the boilerplate is out of date :D @reinismu if you care to provide a commit to fix this it'll be welcome :)

Reply:
Where is this boilerplate? :)

Reply:
well, that volatile bit is boilerplate, copy pasted from somewhere else, just need to be removed really :)